src/fiatconnect/KycLanding.test.tsx

```diff
@@ -60,6 +65,32 @@ describe('KycLanding', () => {
       fireEvent.press(getByTestId('checkbox'))
       expect(getByTestId('PersonaButton')).not.toBeDisabled()
     })
+    it('passes a callback function for successful persona result', () => {
+      const mockPersonaButton = jest.fn(() => <View></View>)
+      jest.spyOn(Persona, 'default').mockImplementation(mockPersonaButton)
+      render(
+        <Provider store={store}>
+          <KycLanding {...props} />
+        </Provider>
+      )
+      expect(Persona.default).toHaveBeenCalledWith(
+        expect.objectContaining({ onSuccess: expect.any(Function) }), // props for Persona
+        expect.anything()
+      )
+      jest.restoreAllMocks()
```

Comment:
You can use `mockImplementationOnce` above and avoid this restoreAll or just explicitly restore the specific mock, as this could potentially lead to issues in case we have some global mock shared among tests.